### PR TITLE
Adding sumo fields which are required by the s3m otel apps to work

### PR DESCRIPTION
**Pull request by sumoanema**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md [hunk 1]
```diff
@@ -23,9 +23,10 @@ We use the Sumo Logic Distribution for OpenTelemetry Collector to collect Linux
 
 Following are the [fields](/docs/manage/fields) that will be created as part of Linux app install if not already present. 
 
-### Across apps
-
 - **`sumo.datasource`** - has a fixed value of **linux**.
+- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation).Through this linux host cluster is identified by the environment where it resides. For example: dev, prod or qa.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md:27]:
```suggestion
- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation). Through this linux host cluster is identified by the environment where it resides. For example: dev, prod, or qa.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md [hunk 2]
```diff
@@ -23,9 +23,10 @@ We use the Sumo Logic Distribution for OpenTelemetry Collector to collect Linux
...
+- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation).Through this linux host cluster is identified.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md:28]:
```suggestion
- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation). Through this linux host cluster is identified.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md [hunk 3]
```diff
@@ -26,6 +26,9 @@ The OpenTelemetry collector runs on the macOS machine, and uses the [Host Metric
 Following are the [fields](/docs/manage/fields/) which will be created as part of the macOS app install, if not already present. 
 
 - **`sumo.datasource`** - Has fixed value of **mac**.
+- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation).Through this mac host cluster is identified by the environment where it resides. For example: dev, prod or qa.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md:29]:
```suggestion
- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation). Through this mac host cluster is identified by the environment where it resides. For example: dev, prod, or qa.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md [hunk 4]
```diff
@@ -26,6 +26,9 @@ The OpenTelemetry collector runs on the macOS machine, and uses the [Host Metric
...
+- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation).Through this mac host cluster is identified.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md:30]:
```suggestion
- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation). Through this mac host cluster is identified.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 5]
```diff
@@ -25,6 +25,9 @@ The Windows app, which is based on the Windows event log format, consists of pre
 Following are the [fields](/docs/manage/fields/) which will be created as part of Windows app install if not already present. 
 
 - **`sumo.datasource`**. Has a fixed value of **windows**.
+- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation).Through this windows host cluster is identified by the environment where it resides. For example: dev, prod or qa.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:28]:
```suggestion
- **`deployment.environment`**. This is a collector level field and is user configured(at the time of collector installation). Through this windows host cluster is identified by the environment where it resides. For example: dev, prod, or qa.
```

File: docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md [hunk 6]
```diff
@@ -25,6 +25,9 @@ The Windows app, which is based on the Windows event log format, consists of pre
...
+- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation).Through this windows host group is identified.
```
Comment (JV0812) [inline docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md:29]:
```suggestion
- **`host.group`**. This is a collector level field and is user configured(at the time of collector installation). Through this windows host group is identified.
```

**Review by jpipkin1 — APPROVED** [review]
@sumoanema, I approved the PR. I made a few minor updates (see commits "[Updates per review](https://github.com/SumoLogic/sumologic-documentation/pull/4043/commits/abd42783bd8e3dbaedfcc100bdb1b41da8557641)" and "[More updates per review](https://github.com/SumoLogic/sumologic-documentation/pull/4043/commits/8b19eaf89a4afc5ea956581ec42b035b9dc311a6)").

**Pull request merged**